### ISSUE TITLE
SPI not implemented

### ISSUE THREAD

**Issue by brennen** [adafruit/Adafruit_CircuitPython_BME680#10]
With latest release from PyPI, on Raspberry Pi 3 model B, using example code from [the BME680 guide](https://learn.adafruit.com/adafruit-bme680-humidity-temperature-barometic-pressure-voc-gas/circuitpython-wiring-test):

```
(blinkenlight) pi@raspberrypi:~/blinkenlight $ cat bme680_spi.py 
import board
import busio
import digitalio
import time
 
import adafruit_bme680

spi = busio.SPI(board.SCK, MOSI=board.MOSI, MISO=board.MISO)
cs = digitalio.DigitalInOut(board.D5)
sensor = adafruit_bme680.Adafruit_BME680_SPI(spi, cs)
 
# Main loop prints sensor values every second.
while True:
    print('Temperature: {} degrees C'.format(sensor.temperature))
    print('Gas: {} ohms'.format(sensor.gas))
    print('Humidity: {}%'.format(sensor.humidity))
    print('Pressure: {}hPa'.format(sensor.pressure))
    time.sleep(1.0)

(blinkenlight) pi@raspberrypi:~/blinkenlight $ python3 bme680_spi.py 
Traceback (most recent call last):
  File "bme680_spi.py", line 10, in <module>
      sensor = adafruit_bme680.Adafruit_BME680_SPI(spi, cs)
      AttributeError: module 'adafruit_bme680' has no attribute 'Adafruit_BME680_SPI'
```

**Comment by ladyada** [comment]
yah its not implemented

**Comment by makermelissa** [comment]
Fixed with #16.
